app/code/Andrey/ServiceContract/Controller/Index/Index.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Andrey\ServiceContract\Controller\Index;
+
+use Magento\Framework\App\Action\Context;
+use Magento\Framework\View\Result\PageFactory;
+use Magento\Framework\App\Action\Action;
+
```

Comment:
add this `use Magento\Framework\Controller\ResultFactory;`
and this `use Magento\Framework\Controller\ResultInterface;`